potodo/_github.py

```diff
@@ -1,32 +1,45 @@
 import requests
 
-from subprocess import getoutput
+from subprocess import check_output
 
 
-def get_repo_url():
-    url = getoutput("git remote get-url --all upstream")
+def get_repo_url() -> str:
+    """
+    Tries to get the repository url from git commands
+    """
+    url: str = str(check_output("git remote get-url --all upstream"))
     if "fatal" in url:
-        url = getoutput("git remote get-url --all origin")
+        url = str(check_output("git remote get-url --all origin"))
     if "fatal" in url:
+        # If the commands didn't work
         raise ValueError(
             f"Unknown error. `git get-url --all upstream|origin` returned {url}"
         )
     return url
 
 
-def get_repo_name():
+def get_repo_name() -> str:
+    """
+    Will get the repository url from git commands then remove useless stuff to get ORG/NAME
+    """
     repo_url: str = get_repo_url()
-    repo_url = repo_url.replace("https://github.com/", "")
-    repo_url = repo_url.replace("git@github.com:", "")
-    repo_name = repo_url.replace(".git", "")
+    # Removes useless stuff. If it isn't there then nothing happens
+    repo_url = repo_url.strip("https://github.com/")
+    repo_url = repo_url.strip("git@github.com:")
+    repo_name = repo_url.strip(".git")
```

Comment:
This won't work as `replace`, it will just strip any chars in the `set(https://github.com/""git@github.com:"".git")`, namely: `{'h', 'b', 'o', '.', 't', ':', 'g', '/', '@', 'c', 'm', 's', 'u', 'p', 'i'}` so:
```
>>> repo_url = "bogus ghotics"
>>> repo_url = repo_url.strip("https://github.com/")
>>> repo_url = repo_url.strip("git@github.com:")
>>> repo_name = repo_url.strip(".git")
>>> repo_name
' '
```

Reply:
So is strip ok ?

Reply:
I don't think so, let me provide an other example:
```
>>> repo_url = "https://github.com/bogus/ghotics.git"
>>> repo_url = repo_url.strip("https://github.com/")
>>> repo_url = repo_url.strip("git@github.com:")
>>> repo_name = repo_url.strip(".git")
>>> repo_name
''
```

#sorryGothics